src/main/java/Main.java

```diff
@@ -233,33 +233,6 @@ private static void _main(String[] args) throws Exception {
             }
         }
 
-        // if we need to daemonize, do it first
-        for (int i = 0; i < args.length; i++) {
-            if(args[i].startsWith("--daemon")) {
-                Map<String, String> revisions = parseDependencyVersions();
```

Comment:
So we can also delete `parseDependencyVersions` and `DEPENDENCIES_LIST`.

Reply:
Done